examples/analog_in/cont_voltage_acq_int_clk.py

```diff
@@ -0,0 +1,31 @@
+﻿"""Example of analog input voltage acquisition.
+
+This example demonstrates how to acquire a continuous amount of data
+using the DAQ device's internal clock.
+"""
+
+import pprint
+import time
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+pp = pprint.PrettyPrinter(indent=4, compact=True)
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_voltage_chan("Dev1/ai0")
+    task.timing.cfg_samp_clk_timing(
+        1000.0, sample_mode=AcquisitionType.CONTINUOUS, samps_per_chan=1000
+    )
+    task.start()
+
+    try:
+        while True:
+            data = task.read(number_of_samples_per_channel=100)
+            print(f"Acquired data: {pp.pformat(data)}")
+            print("Press Ctrl+C to stop.")
```

Comment:
A single "acquired data" print doesn't fit in the default-sized terminal window.

![image](https://github.com/ni/nidaqmx-python/assets/17914399/29faf1ec-3402-4a68-b094-91ffd52b1226)

Reply:
Thanks for trying it, I think I will reduce the `samps_per_chan`.